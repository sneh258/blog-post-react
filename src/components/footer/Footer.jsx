import "./footer.css";

export default function Footer() {
  return (
    <div className="footer">
      <footer>
        <div id="footercontent">
          <h5>&copy;artifact.com 2019</h5>
        </div>
      </footer>
    </div>
  );
}
