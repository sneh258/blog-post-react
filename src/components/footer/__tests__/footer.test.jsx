import React from "react";
import Footer from './../Footer';
import {render } from "@testing-library/react";

describe("Header", () => {
    it("should render correctly", () => {
      const { asFragment } = render(<Footer />);
      expect(asFragment()).toMatchSnapshot();
    });
})