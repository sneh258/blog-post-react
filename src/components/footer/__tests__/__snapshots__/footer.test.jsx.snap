// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render correctly 1`] = `
<DocumentFragment>
  <div
    class="footer"
  >
    <footer>
      <div
        id="footercontent"
      >
        <h5>
          Â©artifact.com 2019
        </h5>
      </div>
    </footer>
  </div>
</DocumentFragment>
`;
